zio-config/src/main/scala/zio/config/actions/ConfigManPage.scala

```diff
@@ -2,27 +2,27 @@ package zio.config.actions
 
 import zio.config.ConfigDescriptor.Succeed
 import zio.config.ConfigDescriptor
-import zio.config.actions.UserManual.KeyDescription
+import zio.config.actions.ConfigManPage.KeyDescription
 
 /**
  * A config description is a description of all keys, and the description of another config and the description of...
  * where only one of them need to exist for successful parsing (Config.Or).
  */
-final case class UserManual(keyDescriptions: List[KeyDescription], or: Option[UserManual])
+final case class ConfigManPage(configKeysAndDescription: List[KeyDescription], or: Option[ConfigManPage])
```

Comment:
I might call this, `ConfigDocumentation` or shorten it to `ConfigDocs`

Reply:
Done